src/Microsoft.TestPlatform.CrossPlatEngine/DataCollection/InProcDataCollectionExtensionManager.cs

```diff
@@ -166,6 +161,7 @@ private void TriggerUpdateTestResult(object sender, TestResultEventArgs e)
         {
             // Just set the cached in-proc data if already exists
             this.SetInProcDataCollectionDataInTestResult(e.TestResult);
+            ((InProcDataCollectionSink)this.inProcDataCollectionSink).RemoveDataCollectionDataSetForTestCase(e.TestCaseId);
```

Comment:
This is a bug that we introduced as part of previous refactoring. Due to this bug, Cross and Check were not visible in Code Editor when LUT was enabled.